src/OSPSuite.Presentation/Mappers/ParameterIdentifications/ParameterIdentificationRunResultToRunResultDTOMapper.cs

```diff
@@ -56,16 +56,16 @@ private OptimizedParameterDTO mapFrom(OptimizedParameterValue optimizedParameter
          if (identificationParameter == null)
             return null;
 
-         return mapFrom(identificationParameter, optimizedParameterValue.Value);
+         return mapFrom(identificationParameter, optimizedParameterValue.Value, optimizedParameterValue.StartValue);
       }
 
-      private OptimizedParameterDTO mapFrom(IdentificationParameter identificationParameter, double optimalValue)
+      private OptimizedParameterDTO mapFrom(IdentificationParameter identificationParameter, double optimalValue, double startValue)
       {
          var dto = new OptimizedParameterDTO
          {
             Name = identificationParameter.Name,
             OptimalValue = mapFrom(optimalValue, identificationParameter.StartValueParameter),
-            StartValue = mapFrom(identificationParameter.StartValue, identificationParameter.StartValueParameter),
```

Comment:
We were always taking the start value from the IdentificationParameter object instead of the actual start value of the result